docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -93,6 +93,44 @@ For more details on the Keycloak upgrade path, you can also read the [Bitnami Ke
 
 ## Version update instructions
 
+### v8.3
+
+:::caution Breaking change
+
+Zeebe now runs as a non-root user by default.
+
+:::
+
+Using a non-root user by default is a security principle introduced in this version. However, because there is persistent storage in Zeebe, earlier versions may run into problems with existing file permissions not matching up with the file permissions assigned to the running user. There are two ways to fix this:
+
+1. (Recommended) Change the `podSecurityContext fsGroup` to point to the UID of the running user. The default user in Zeebe has the UID 1001. `fsGroup` will modify the group permissions of all persistent volumes attached to that pod.
```

Comment:
:question: Is this enough? I understand it will change the permission of the mount point, but will it also change the permission of all files/folders in the volume as well? Did we verify this?

Reply:
If I understand https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods correctly it is.

```
By default, Kubernetes recursively changes ownership and permissions for the contents of each volume to match the fsGroup specified in a Pod's securityContext when that volume is mounted. 
```